comparisons/navigation/accordion/html.html

```diff
@@ -1,19 +1,19 @@
-<div class="togglebox">
+<div class="togglebox" role="tablist">
```

Comment:
I tested it on NVDA & Chromevox and it's work correctly with a keyboard if you used tab and up,down arrow will hear and see the content.